agot-bg-game-server/src/client/IngameComponent.tsx

```diff
@@ -574,4 +575,18 @@ export default class IngameComponent extends Component<IngameComponentProps> {
             <VoteComponent key={v.id} vote={v} gameClient={this.props.gameClient} ingame={this.props.gameState} />
         ));
     }
+
+    componentDidMount(): void {
+        this.props.gameState.entireGame.onNewPrivateChatRoomCreated = (chatInitiator: User, roomId: string) => this.onNewPrivateChatRoomCreated(chatInitiator, roomId);
+    }
+
+    onNewPrivateChatRoomCreated(chatInitiator: User, roomId: string): void {
+        if (this.props.gameClient.isAuthenticatedUser(chatInitiator)) {
```

Comment:
@Longwelwind This line doesn't do the trick?

Reply:
No, because `users[0]` is not necessarily the party that initiated the private chat. It will be the first user in the private room, by alphabetical order of id.